platform/security/platform-security-core-api/src/main/java/ddf/security/SubjectUtils.java

```diff
@@ -162,6 +163,30 @@ public static String getCommonName(X500Principal principal) {
                 .toString();
     }
 
+    public static String getEmailAddress(X500Principal principal) {
+        RDN[] rdNs = new X500Name(principal.getName()).getRDNs(BCStyle.EmailAddress);
+        if (rdNs != null && rdNs.length > 0) {
+            AttributeTypeAndValue attributeTypeAndValue = rdNs[0].getFirst();
+            if (attributeTypeAndValue != null) {
+                return attributeTypeAndValue.getValue()
+                        .toString();
+            }
+        }
+        return null;
+    }
+
+    public static String getCountry(X500Principal principal) {
```

Comment:
These two methods are almost identical except for the `BCStyle` consts. Could refactor. Only really worth it if we ever think we might want to extract any more string attributes in this way.